diff --git a/node_modules/@vitejs/plugin-react/dist/index.d.ts b/node_modules/@vitejs/plugin-react/dist/index.d.ts
index 2e7c63d..753aee3 100644
--- a/node_modules/@vitejs/plugin-react/dist/index.d.ts
+++ b/node_modules/@vitejs/plugin-react/dist/index.d.ts
@@ -72,4 +72,5 @@ declare namespace viteReact {
 }
 declare function viteReactForCjs(this: unknown, options: Options): Plugin[];
 //#endregion
-export { BabelOptions, Options, ReactBabelOptions, ViteReactPluginApi, viteReact as default, viteReactForCjs as "module.exports" };
\ No newline at end of file
+export { BabelOptions, Options, ReactBabelOptions, ViteReactPluginApi, viteReact as default };
+export { viteReactForCjs };
\ No newline at end of file
